.repo-metadata.json

```diff
@@ -6,6 +6,7 @@
   "issue_tracker": "https://issuetracker.google.com/savedsearches/559749",
   "release_level": "ga",
   "language": "python",
+  "library_type": "GAPIC_AUTO",
```

Comment:
One note, the v3 is GAPIC_AUTO, v2 is handwritten (https://github.com/googleapis/python-translate/tree/master/google/cloud/translate_v2). Since one value has to be chosen I agree with labeling this as GAPIC_AUTO.